gitensite/apps/bookinfo/management/commands/load_repos.py

```diff
@@ -26,10 +26,13 @@ def handle(self, rdf_library, start=0, *args, **kwargs):
             if int(pg_id)<start:
                 continue
             try:
-                metadata=BookMetadata(Book(book_id=pg_id), rdf_library=rdf_library, enrich=should_enrich)    
-                addBookFromYaml(metadata.__unicode__())
+                metadata=BookMetadata(Book(book_id=pg_id), rdf_library=rdf_library, enrich=should_enrich)
+                
+                # Add repo_name to yaml
+                yaml = metadata.__unicode__() + "\nrepo_name: " + repo_name
```

Comment:
this is error-prone in a number of ways. Better  to do
metadata.metadata["_repo"] = repo_name